libraries/BLE/src/ArduinoBLE.h

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
@noelpaz right.

Here's a proposal for how we should move forward in this fork/PR for now. Since @sgbihu has added the backwards compatible layer.

1) Delete `libraries/CurieBLE`
2) Rename `libraries/BLE` to `libraries/CurieBLE`
3) Create a new ` `libraries/CurieBLE/examples/deprecated` folder and copy the example sketches from core release v1.0.7

We can remove the deprecated example sketches prior to releasing.